pipeline-steps/sql-access.yaml

```diff
@@ -0,0 +1,32 @@
+parameters:
```

Comment:
Do we need this, can we not just use ?

https://registry.terraform.io/providers/betr-io/mssql/latest/docs/resources/user